src/js/rx/routes.jsx

```diff
@@ -12,10 +12,10 @@ const routes = {
       component: Main,
       indexRoute: { component: Active },
       childRoutes: [
-        { path: '/history', component: History }
+        { path: '/rx/history', component: History }
```

Comment:
Turns out these don't need the `/rx' or even the leading slash, so you can leave those out.